pkg/iostreams/color.go

```diff
@@ -9,22 +9,21 @@ import (
 )
 
 var (
-	magenta   = ansi.ColorFunc("magenta")
-	cyan      = ansi.ColorFunc("cyan")
-	red       = ansi.ColorFunc("red")
-	yellow    = ansi.ColorFunc("yellow")
-	blue      = ansi.ColorFunc("blue")
-	green     = ansi.ColorFunc("green")
-	gray      = ansi.ColorFunc("black+h")
-	bold      = ansi.ColorFunc("default+b")
-	cyanBold  = ansi.ColorFunc("cyan+b")
-	greenBold = ansi.ColorFunc("green+b")
+	magenta       = ansi.ColorFunc("magenta")
+	cyan          = ansi.ColorFunc("cyan")
+	red           = ansi.ColorFunc("red")
+	yellow        = ansi.ColorFunc("yellow")
+	blue          = ansi.ColorFunc("blue")
+	green         = ansi.ColorFunc("green")
+	gray          = ansi.ColorFunc("black+h")
+	grayUnderline = ansi.ColorFunc("white+du")
```

Comment:
Nit: @samcoe would you want to call this `lightGrayUnderline`? It's slightly lighter than `gray` (depending on terminal color mappings). That's why I decided to originally call the style `TableHeader` (not complaining you changed it, mind you): it's the obvious style for a table header and doesn't get too specific about the colors.

A few styles are called / include "gray" here and are all slightly different colors.

To note, I choose "white+du" over "black+hu" or "black+bu" because it was slightly different than gray that's used for various columns e.g. dates. Not hugely noticeable - at least not with Windows Terminal's default color mappings - and maybe unnecessary now that we have the underlines.

Reply:
That's a good point, will make that change before merging.